tests/unittests/OnnxImporterTest.cpp

```diff
@@ -199,6 +199,72 @@ TEST(onnx, importUniBroadcastMultiOutput) {
   (void)onnxLD;
 }
 
+static void testImportPRelu(std::string filename,
+                            llvm::ArrayRef<size_t> inputShape,
+                            std::vector<float> expectedSlope) {
+  ExecutionEngine EE{BackendKind::Interpreter};
+  auto &mod = EE.getModule();
+  Function *F = mod.createFunction("main");
+
+  std::string NetFileName =
+      std::string(GLOW_DATA_PATH "tests/models/onnxModels/") + filename;
+
+  PlaceholderBindings bindings;
+  Placeholder *graphOutputVar;
+  // Destroy the loader after the graph is loaded since the following execution
+  // will not depend on anyting from the loader.
+  {
+    Tensor data(ElemKind::FloatTy, inputShape);
+    data.getHandle().randomize(-4.0, 4.0, mod.getPRNG());
+    ONNXModelLoader onnxLoader(NetFileName, {"data"}, {&data.getType()}, *F);
+    graphOutputVar = EXIT_ON_ERR(onnxLoader.getSingleOutput());
+    bindings.allocate(mod.getPlaceholders());
+    updateInputPlaceholdersByName(bindings, &mod, {"data"}, {&data});
+  }
+
+  // Compile&run the graph, and check the output.
+  EE.compile(CompilationMode::Infer, F);
+  EE.run(bindings);
+  auto dataH = bindings.get(bindings.getPlaceholderByName("data"))->getHandle();
+  auto result = bindings.get(graphOutputVar)->getHandle();
+  std::vector<size_t> expectedDims = {inputShape[0], inputShape[1],
+                                      inputShape[2], inputShape[3]};
+
+  EXPECT_TRUE(result.dims().vec() == expectedDims);
+  for (size_t i = 0; i < dataH.size(); i++) {
+    float expectedVal = expectedSlope[i] * std::min<float>(0, dataH.raw(i)) +
+                        std::max<float>(0, dataH.raw(i));
+    EXPECT_FLOAT_EQ(result.raw(i), expectedVal);
```

Comment:
typically for those kind of tests we just verify the graph structure and make it done. (this is already tested in operator test)

Reply:
talked offline, let's do computation check here due to complex reshape/tile sequences.